libraries/esp8266/examples/HwdtStackDump/hwdt_app_entry.cpp

```diff
@@ -334,7 +319,7 @@ extern uint32_t stack_thunk_get_stack_bot() __attribute__((weak, alias("__zero_r
 #if !defined(HWDT_STACK_DUMP_H) || defined(HWDT_VERIFY_HWDT_INFO)
 #define HWDT_STACK_DUMP_H
 
-typedef struct HWDT_INFO {
+typedef struct HWDT_INFO_S {
```

Comment:
(minor style nitpick)
- since this is an example, to we still need to have `typedef struct`? yes, c++ still supports it, but we can just do `struct name { ... };`
- should the struct name be `HwdtInfo` to avoid confusion with UPPERCASE macros?

Reply:
@mcspr thanks for the suggestion. 

I agree it is best to avoid confusion with UPPERCASE for macros. My intent was to offer an example with a self-contained debug module(hwdt_app_entry.cpp) that would in effect be supplementing the core for debugging purposes. The example files use camel-case and to set it apart, the debug module uses snake-case, similar to most of the core files I see.  So for the structure, I have matched the style to that I see in the core files.

The intent was that `hwdt_app_entry.cpp` would be more a portable module, not an example.
With the exception of some user-configurable `#defines` at the top of `hwdt_app_entry.cpp`, there should be little need to change the code. A least that was my goal.

Reply:
Thanks for the clarification!

I just noticed the
> A strange bug appeared I discovered I duplicated the name HWDR_INFO. I had it as a #define and a structure name. 

Which is cleared up by the commit after this one. I rest my case :)